game/scripts/vscripts/modifiers/modifier_sonic_fly.lua

```diff
@@ -0,0 +1,24 @@
+modifier_sonic_fly = class(ModifierBaseClass)
+
+function modifier_sonic_fly:DeclareFunctions()
+  return {
+      MODIFIER_PROPERTY_MOVESPEED_BONUS_CONSTANT,
+      MODIFIER_PROPERTY_BONUS_VISION_PERCENTAGE,
+      MODIFIER_PROPERTY_STATUS_RESISTANCE
```

Comment:
Have you tested if `MODIFIER_PROPERTY_STATUS_RESISTANCE` works? I hear rumors....

Reply:
Upon further testing and research I am unable to find a way to change the status resistance